Frontend-Angular/src/app/app.component.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { TestBed } from '@angular/core/testing';
+import { AppComponent } from './app.component';
```

Comment:
we can remove the unit tests and let the candidate decide? this will be plus factor if they add tests